app/interactors/rake_interactor.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'rake'
+
+class RakeInteractor
+  include Interactor
+
+  def initialize(task_name: '', args: {}, log_file: nil)
+    @task_name = task_name
+    @args      = args
+    @log_file  = log_file || "#{Rails.root}/log/rake.log"
```

Comment:
In case we don't want any particular log file, I defaulted this to /log/rake.log